src/pages/Stream/providers/LogsProvider.tsx

```diff
@@ -260,7 +260,12 @@ type LogsStoreReducers = {
 	getCleanStoreForRefetch: (store: LogsStore) => ReducerOutput;
 
 	// data reducers
-	setData: (store: LogsStore, data: Log[], schema: LogStreamSchemaData | null, jqFilteredData?: Log[]) => ReducerOutput;
+	setData: (
+		store: LogsStore,
+		data: Log[],
+		queryResponse: LogStreamQueryWithFields | null,
```

Comment:
```
data: Log[],
queryResponse: LogStreamQueryWithFields | null
```

With logs provider context, both deal with almost the same values. Rmv one with less significance.